doc/man1/flux-exec.rst

```diff
@@ -52,21 +53,27 @@ OPTIONS
 
 .. option:: -r, --rank=IDSET
 
-   Target specific ranks in *IDSET*. Default is to target "all" ranks.
+   Target specific ranks, where *IDSET* is a set of zero-origin node ranks in
+   RFC 22 format.  If :option:`--jobid` is specified, the ranks are interpreted
+   as an index into the list of nodes assigned to the job.  Otherwise, they
+   refer to the nodes assigned to the Flux instance.
+
+   The default is to target all ranks.
 
 .. option:: -x, --exclude=IDSET
 
-   Exclude ranks in *IDSET*.
+   Exclude specific ranks.  *IDSET* is as described in :option:`--rank`.
 
 .. option:: -j, --jobid=JOBID
 
-   Use the job shell exec service for job *JOBID* instead of the broker's
-   exec service. The default *IDSET* will be set to the nodes assigned
-   to the job, and the :option:`--rank` and :option:`--exclude` options are
-   applied relative to this set. For example, :option:`flux exec -j ID -r 0`
-   will run only on the first node assigned to *JOBID*, and :option:`flux
-   exec -j ID -x 0` will run on all nodes assigned to *JOBID* except the
-   first node.
+   Run *COMMAND* on the nodes allocated to *JOBID* instead of the nodes
+   assigned to the Flux instance.
```

Comment:
This drops the note that he job shell exec service is used instead of the builtin broker exec service. Should something to that effect be included -- at least a hint  that this gives guests the ability to use `flux exec` against their jobs in a multiuser instance?

Reply:
Oh good point.  I just pushed a small fixup that I think clarifies this?

Reply:
Yes, that's great! Feel free to squash.